tests/main/interfaces-audio-playback-record/task.yaml

```diff
@@ -0,0 +1,108 @@
+summary: Ensure that the audio-playback/record interface works
+
+# Only test on classic Ubuntu amd64 systems
```

Comment:
Rather than saying what the subsequent line does, it would be better to say why. I suspect it is because we don't have the appropriate patches anywhere else but I don't know.